src/app/routeValidator/helpers/validateRouteParameter.test.js

```diff
@@ -0,0 +1,21 @@
+import validateRouteParameter from './validateRouteParameter';
+
+// global.console.log = jest.fn(); // silence console.log during jest testss
```

Comment:
Please remove your commented line

Reply:
`// silence console.log during jest testss` or the whole line?